.travis.yml

```diff
@@ -4,4 +4,4 @@ install: gem install asciidoctor
 script: "./tools/convert.sh"
 notifications:
   email:
-    - alexander.grueneberg@googlemail.com
+    - github@mikehardy.net
```

Comment:
@mikehardy, thanks for updating this. I can't remember ever receiving any emails for this pipeline except from forks that don't have permission to perform the update. There are not many, though.